deployment/docker-compose/docker-compose.yml

```diff
@@ -8,7 +8,11 @@ services:
       /bin/sh startup.sh
     restart: always
     env_file:
-      - .env
+      - .common.env
+      - .core_backend.env
+    environment:
+      - REDIS_HOST=redis://redis:6379
+      - LITELLM_ENDPOINT=http://litellm_proxy:4000
```

Comment:
I've instructed in template that these should be updated for docker compose & for production.